lib/cc/presenters/github_pull_requests_presenter.rb

```diff
@@ -0,0 +1,62 @@
+module CC
```

Comment:
This file was just moved from `lib/cc/services` to a new `lib/cc/presenters` directory, and nested under `module CC` and `class Service`, which less susceptible to load order issues than doing `class CC::Service::GithubPullRequestsPresenter`